cmsmadesimple/caddy@.service

```diff
@@ -0,0 +1,32 @@
+; see `man systemd.unit` for configuration details
+; the man section also explains *specifiers* `%x`
+
+[Unit]
+Description=Caddy HTTP/2 web server %I
+Documentation=https://caddyserver.com/docs
+After=network-online.target
+Wants=network-online.target
+Wants=systemd-networkd-wait-online.service
+
+[Service]
+; run user and group for caddy
+User=username
+Group=username
+ExecStart=/usr/local/bin/caddy -conf=/home/username/caddy/Caddyfile -agree -email="firstname.lastname@example.com"
+Restart=on-failure
+RestartSec=10
```

Comment:
I'm told this will fix it: https://github.com/mholt/caddy/commit/466269fd10e2e9a2fb70ef80609dd379a44fbd4b